src/script/components/MessagesList/Message/ContentMessage/ContentMessage.tsx

```diff
@@ -184,7 +184,7 @@ export const ContentMessageComponent = ({
         className={cx('message-body', {
           'message-asset': isAssetMessage,
           'message-quoted': !!quote,
-          'ephemeral-asset-expired': isObfuscated,
+          'ephemeral-asset-expired': isObfuscated && isAssetMessage,
```

Comment:
By looking at the diff [here](https://github.com/wireapp/wire-webapp/pull/16887/files#diff-3f9b5143b3065b819bb2706445ea4f06a160b30ab205f034543be319d47166efL125-L165), you can see that that class only applied to asset messages before the change